test/docker_test.rb

```diff
@@ -1,55 +0,0 @@
-require_relative "docker_run"
```

Comment:
We do still create a docker image during release - and some people are consuming it, apparently (we had a recent issue that mentioned it indirectly).

Reply:
Sure. But we don't run any of these `test/docker*` files anywhere that I can find.